src/System.Windows.Forms/src/System/Windows/Forms/ButtonBase.ButtonBaseAccessibleObject.cs

```diff
@@ -26,5 +26,43 @@ public override void DoDefaultAction()
                 owner.OnClick(EventArgs.Empty);
             }
         }
+
+        internal static string? GetKeyboardShortcut(Control control, bool useMnemonic, Label? previousLabel)
+        {
+            char mnemonic = '\0';
+
+            if ((!useMnemonic || !WindowsFormsUtils.ContainsMnemonic(control?.Text)) && previousLabel is not null && previousLabel.UseMnemonic)
+            {
+                mnemonic = WindowsFormsUtils.GetMnemonic(previousLabel.Text, false);
```

Comment:
Could you also specify parameter name for `GetMnemonic` calls?
```suggestion
                mnemonic = WindowsFormsUtils.GetMnemonic(previousLabel.Text, convertToUpperCase: false);
```